extensions/amp-carousel/0.1/test-e2e/test-slidescroll-autoplay.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describes.endtoend(
+  'AMP carousel 0.1 slideChange on type="slide" with autoplay',
+  {
+    testUrl:
+      'http://localhost:8000/test/fixtures/e2e/amp-carousel/0.1/slidescroll-autoplay.html',
+    environments: ['single'],
+  },
+  async function(env) {
+    const ActionTrust = {LOW: 1, HIGH: 3};
```

Comment:
I mentioned this earlier but if we ever need to add an enum between low and default, it might be awkward (non-integer).

Reply:
Pros for exposing strings:

- Human-readable and meaningful, especially in debugging
- Adding intermediary values is straight forward
- Int values are an implementation detail within AMP, exposing them as such also means making others mimic this implementation.

Pros for exposing integers:

- Easier to compare (`trust < high`)
- Not localized to us -- i.e. "low" is in English
- Faster, esp. since trust is checked often in AMP

Reply:
Those look good. We could also change the integer enum values to have some breathing room -- I think they originally did before I removed it. :)

Reply:
What was the reasoning behind removing it?

Reply:
Just a whim. :p 

This most likely won't be necessary since 4 tiers of "trust" is quite a lot to grok. Just wanted to point out the option.

Reply:
SG, going to go ahead and merge in this case.